src/common.ts

```diff
@@ -1,18 +1,40 @@
-export interface TransactionOptions {
-  readOnly?: boolean;
-  timeout?: number;
-  exactStaleness?: number;
-  readTimestamp?: Date;
-  returnTimestamp?: boolean;
-  strong?: boolean;
+import {google as databaseAdmin} from '../proto/spanner_database_admin';
+import {ServiceError, CallOptions} from 'grpc';
+
+export type Schema =
+  | string
+  | string[]
+  | databaseAdmin.spanner.admin.database.v1.IUpdateDatabaseDdlRequest;
+export type ResourceCallback<Resource, Response = void> = Response extends void
```

Comment:
Why are we conditionally changing the number of arguments here?

Reply:
added `Response` required type and used `NormalCallback<Resource>`  instead `ResourceCallback<Resource>`